libandroid-navigation/src/main/java/org/maplibre/navigation/android/navigation/v5/navigation/MapLibreNavigationOptions.java

```diff
@@ -1,162 +0,0 @@
-
-  public abstract int secondsBeforeReroute();
-
-  public abstract boolean defaultMilestonesEnabled();
-
-  public abstract boolean snapToRoute();
-
-  public abstract boolean enableOffRouteDetection();
-
-  public abstract boolean enableFasterRouteDetection();
-
-  public abstract boolean manuallyEndNavigationUponCompletion();
-
-  public abstract double metersRemainingTillArrival();
-
-  public abstract boolean isFromNavigationUi();
-
-  public abstract double minimumDistanceBeforeRerouting();
-
-  /**
-   * Minimum distance in meters that the user must travel in the wrong direction before the
-   * off-route logic recognizes the user is moving away from upcoming maneuver
-   */
-  public abstract double offRouteMinimumDistanceMetersBeforeWrongDirection();
-
-  /**
-   * Minimum distance in meters that the user must travel in the correct direction before the
-   * off-route logic recognizes the user is back on the right direction
-   */
-  public abstract double offRouteMinimumDistanceMetersBeforeRightDirection();
-
-  public abstract boolean isDebugLoggingEnabled();
-
-  @Nullable
-  public abstract NavigationNotification navigationNotification();
-
-  @NavigationConstants.RoundingIncrement
-  public abstract int roundingIncrement();
-
-  @NavigationTimeFormat.Type
-  public abstract int timeFormatType();
-
-  public abstract int locationAcceptableAccuracyInMetersThreshold();
-
-  public abstract Builder toBuilder();
-
-  @AutoValue.Builder
-  public abstract static class Builder {
-
-    public abstract Builder maxTurnCompletionOffset(double maxTurnCompletionOffset);
-
-    public abstract Builder maneuverZoneRadius(double maneuverZoneRadius);
-
-    /**
-     * @deprecated has no effect and will be removed in a future release. Use {@link #minimumDistanceBeforeRerouting()} instead.
-     */
-    @Deprecated
-    public abstract Builder maximumDistanceOffRoute(double maximumDistanceOffRoute);
-
-    public abstract Builder deadReckoningTimeInterval(double deadReckoningTimeInterval);
-
-    public abstract Builder maxManipulatedCourseAngle(double maxManipulatedCourseAngle);
-
-    public abstract Builder userLocationSnapDistance(double userLocationSnapDistance);
-
-    public abstract Builder secondsBeforeReroute(int secondsBeforeReroute);
-
-    public abstract Builder defaultMilestonesEnabled(boolean defaultMilestonesEnabled);
-
-    public abstract Builder snapToRoute(boolean snapToRoute);
-
-    public abstract Builder enableOffRouteDetection(boolean enableOffRouteDetection);
-
-    public abstract Builder enableFasterRouteDetection(boolean enableFasterRouteDetection);
-
-    public abstract Builder manuallyEndNavigationUponCompletion(boolean manuallyEndNavigation);
-
-    public abstract Builder metersRemainingTillArrival(double metersRemainingTillArrival);
-
-    public abstract Builder isFromNavigationUi(boolean isFromNavigationUi);
-
-    public abstract Builder minimumDistanceBeforeRerouting(double distanceInMeters);
```

Comment:
Was this prop lost in translation?

Reply:
Sounds this reasonable for you?

Reply:
Yeah, upon checking our own code, your decision makes sense to me.